Tasks/PackerBuild/src/taskInputTemplateVariablesProvider.ts

```diff
@@ -46,7 +46,8 @@ export default class TaskInputTemplateVariablesProvider implements definitions.I
         this._templateVariables.set(constants.TemplateVariableLocationName, taskParameters.location);        
 
         var capturePrefix = tl.getVariable('release.releaseName') || tl.getVariable('build.buildnumber') || "vstscapture";
-        this._templateVariables.set(constants.TemplateVariableCapturePrefixName, capturePrefix);        
+        this._templateVariables.set(constants.TemplateVariableCapturePrefixName, capturePrefix);  
+        this._templateVariables.set(constants.TemplateVariableSkipCleanName, taskParameters.skipTempFileCleanupDuringVMDeprovision.toString());      
```

Comment:
since some template dont'  have it; do we handle it well if the template doesn't require it